bindings/python/setup.py

```diff
@@ -17,28 +20,123 @@ def get_pymongoarrow_version():
     return version['__version__']
 
 
+INSTALL_REQUIRES = [
+    'pyarrow>=3,<3.1',
+    'pymongo>=3.11,<4',
+    'numpy>=1.16.6,<2'
+]
+
+
+SETUP_REQUIRES = [
+    'cython>=0.29<1',
+    'pyarrow>=3,<4',
+    'numpy>=1.16.6,<2',
+    'setuptools>=41'
+]
+
+
+TESTS_REQUIRE = [
+    "pandas"
+]
+
+
+def append_libbson_flags(module):
+    if os.environ.get('USE_STATIC_LIBBSON') == '1':
+        pc_name = 'libbson-static-1.0'
+    else:
+        pc_name = 'libbson-1.0'
+        # https://blog.krzyzanowskim.com/2018/12/05/rpath-what/
+        if platform == "darwin":
+            module.extra_link_args += ["-rpath", "@loader_path"]
+
+    if 'LIBBSON_INSTALL_DIR' in os.environ:
+        libbson_pc_path = os.path.join(
+            os.environ['LIBBSON_INSTALL_DIR'], 'lib', 'pkgconfig',
+            ".".join([pc_name, 'pc']))
+    else:
+        libbson_pc_path = pc_name
+
+    status, output = subprocess.getstatusoutput(
+        "pkg-config --cflags {}".format(libbson_pc_path))
+    if status != 0:
+        warnings.warn(output, UserWarning)
+    for entry in output.split():
+        if entry.startswith('-I'):
+            include_dir = entry.lstrip('-I')
+            module.include_dirs.append(include_dir)
```

Comment:
Instead of parsing the pkg-config output we should just set the CFLAGS/LDFLAGS variables in os.environ.

Reply:
Done.